x-pack/plugins/infra/public/apps/start_app.tsx

```diff
@@ -37,14 +38,16 @@ export async function startApp(libs: InfraFrontendLibs) {
         <ConstateProvider devtools>
           <ReduxStoreProvider store={store}>
             <ApolloProvider client={libs.apolloClient}>
-              <ThemeProvider
-                theme={() => ({
-                  eui: libs.framework.darkMode ? euiDarkVars : euiLightVars,
-                  darkMode: libs.framework.darkMode,
-                })}
-              >
-                <PageRouter history={history} />
-              </ThemeProvider>
+              <ApolloClientContext.Provider value={libs.apolloClient}>
```

Comment:
This additional provider is used to make the client available via the new-style context api, which react-apollo doesn't yet support.